### PR TITLE
correction for pvp_ovsdpdk.yml and creat pvp_dut_new.yml

### PR DESCRIPTION
**Pull request by linup2011**

### CODE REVIEW

File: ansible/trex_settings.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ dut_isolated_cpus: 1,3,5,7,9,11,13,15
 dut_interface_1: enp68s0f0
 dut_interface_2: enp68s0f1
 
+dut_interface_1_pciid: "0000:44:00.0"
```
Comment (ctrautma) [inline ansible/trex_settings.yml:34]: I'll have to go back and change the dpdk pvp script as I was detecting this address on the fly but with your adding of this setting to the settings file I no longer need to do that.  I'll push up a commit later.

File: ansible/pvp_dut_new.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  - name: Start openvswitch
+    service:
+      name: openvswitch
+      state: started
+
+#################
+# Re-config OVS #
+#################
+
+  - name: Delete ovs_pvp_br0 if exist
```
Comment (ctrautma) [inline ansible/pvp_dut_new.yml:76]: Looks like you already do this above.  Is it needed again here?

**Review by ctrautma — COMMENTED** [review]
Looks great,  just a question on one thing I see duplicated and not sure why.

File: ansible/trex_settings.yml (hunk 1 above)
Comment (linup2011) [reply ansible/trex_settings.yml:34]: yes, in pvp_ovsdpdk.yml, as the interface is still using kernel driver, you can get the pciid based on the name. but on pvp_dut.yml, the interface is already in dpdk, I can't get the pciid based on the interface name

Comment (linup2011) [reply ansible/trex_settings.yml:34]: but for pvp_ovsdpdk.yml. we can either keep it as it is now or change the script a little. both are OK. and I can help to change it if needed. how do you think

File: ansible/pvp_dut_new.yml (hunk 2 above)
Comment (linup2011) [reply ansible/pvp_dut_new.yml:76]: yes, really a duplication, jut to confirm twice. How about changing it and create another pull request?

**Pull request merged**